docs(feg): Tier is needed to setup an feg (#12903)
magma/magma: docs/readmes/feg/deploy_install.md

(i.e. <https://controller.url.sample:9443/swagger/v1/ui/>).

Now, create a Federation Network. This is found at `/feg` under the

> **Federation Networks** section. If you have not registered any gateways before you

Reviewer: ```suggestion
**Federation Networks** section. If you have not registered any gateways before, you
```

> must setup a tier under the **Orchestrator** section at `/networks/{network_id}/tiers`.

Reviewer: ```suggestion
must set up a tier under the **Orchestrator** section at `/networks/{network_id}/tiers`.
```

> Then register the gateway under the **Federation Gateway** section at `/feg/{network_id}/gateways` 

Reviewer: ```suggestion
Then, register the gateway under the **Federation Gateway** section at `/feg/{network_id}/gateways` 
```
